migrations/2018-01-08-072612_status_uri/down.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE statuses DROP COLUMN uri RESTRICT;
```

Comment:
what does RESTRICT do here?

Reply:
The opposite of CASCADE. It appears to be the default.

Reply:
@nightpool 
> CASCADE
> Automatically drop objects that depend on the dropped column or constraint (for example, views referencing the column).
> 
> RESTRICT
> Refuse to drop the column or constraint if there are any dependent objects. This is the default behavior.